pytorch_lightning/trainer/configuration_validator.py

```diff
@@ -201,3 +203,12 @@ def __check_training_step_requires_dataloader_iter(self, model: "pl.LightningMod
                     "The model taking a `dataloader_iter` argument in your `training_step` "
                     "is incompatible with `truncated_bptt_steps > 0`."
                 )
+
+    def _check_on_keyboard_interrupt(self) -> None:
+        """Checks if on_keyboard_interrupt is overriden and sends a deprecation warning."""
+        for callback in self.trainer.callbacks:
+            if is_overridden(method_name="on_keyboard_interrupt", instance=callback):
+                rank_zero_deprecation(
+                    "The `on_keyboard_interrupt` callback hook was deprecated in v1.5 and will be removed in v1.7."
+                    "Please use the `on_exception` callback hook instead."
```

Comment:
```suggestion
                    "The `on_keyboard_interrupt` callback hook was deprecated in v1.5 and will be removed in v1.7."
                    " Please use the `on_exception` callback hook instead."
```
some spacing needed after the period